addons/website/views/website_pages_views.xml

```diff
@@ -60,7 +60,7 @@
                         <field name="date_publish" widget="datetime"/>
                     </group>
                     <group>
-                        <field name="visibility"/>
+                        <field name="visibility" required="True"/>
```

Comment:
Can't we make this directly at the field declaration? Maybe that will simplifies the behavior and fix the bug I mention in the review.
Maybe also change "All" to "Public" (as it actually was in the 15.0 page properties)
And also change 'All' technical key to 'public'? I feel it weird to have empty string
```py
 visibility = fields.Selection([('public', 'Public'), ('connected', 'Signed In'), ('restricted_group', 'Restricted Group'), ('password', 'With Password')], default='public', required=True)
 ```
 
 Note that it would need a migration script to change `''` to '`public`'.
 
 Any other idea welcome of course :+1:

Reply:
I do not find any other selection field with an empty string key - if it is not considered as a supported feature then I'll go for the 'public' + migration. Maybe I'll add an error if a selection field has a an option with a false key.
This way I can get rid of my fix of the UI.